manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: Tableau.Reader
+PackageVersion: 2021.2.1
+PackageLocale: en-US
+Publisher: Tableau
+PackageName: Tableau Reader
+PackageUrl: https://www.tableau.com/products/reader
+License: (c) 2003-2021 Tableau Software, LLC.
+LicenseUrl: https://mkt.tableau.com/legal/tableau_eula.pdf
+ShortDescription: Tableau Reader is a free desktop application that you can use to open and interact with data visualisations built in Tableau Desktop
+Moniker: tableau-reader
+Tags:
+- charts
+- graphs
+- business
+- tableau
+Installers:
+- Architecture: x64
+  InstallerType: inno

```

Comment:
```suggestion
  InstallerType: burn
```

Reply:
leaving this as InstallerType inno.
It successfully installs in the background without any popup (apart from the UAC)
With InstallerType burn, it displays the installation windows which is undesired

Reply:
huh, is this specific to this installer or does /SILENT and /VERYSILENT work for Burn?